dnf-docker-test/launch-test

```diff
@@ -10,4 +16,11 @@ if [ "$EXIT_STATUS" = 1 ]; then
     >&2 echo "Feature file ${1}.feature not found"
     exit 1
 fi
-behave-2 -i $new_name -D dnf_cmd=$2 --junit --junit-directory /junit/ /behave/
+
+TEST_EXIT=0
```

Comment:
I believe it is due to the bash settings at the beginning of the script.

-u      Treat unset variables and parameters other than the spe-
                      cial  parameters "@" and "*" as an error when performing
                      parameter expansion.  If expansion is  attempted  on  an
                      unset  variable  or parameter, the shell prints an error
                      message, and, if not interactive, exits with a  non-zero
                      status.